kinto_remote_settings/signer/tests/test_signoff_flow.py

```diff
@@ -990,12 +986,19 @@ def setUp(self):
             headers=self.headers,
         )
 
-    def test_editors_can_ask_to_review_if_not_specificly_configured(self):
-        self.app.patch_json(
+    def test_editors_cannot_ask_to_review_if_not_specificly_configured(self):
```

Comment:
Just a small spelling error

```suggestion
    def test_editors_cannot_ask_to_review_if_not_specifically_configured(self):
```